docs/local_exporter.py

```diff
@@ -47,7 +47,6 @@ def download(url, target_file_path):
             request = Request(url, headers={'User-Agent': 'Mozilla'})
             if platform.system() == 'Linux' and \
                hasattr(ssl, 'create_default_context'):
-                # On Ubuntu 16.04 there are issues with the certificates.
```

Comment:
I would keep this comment because it explains why on Linux the behavior is different.